scripts/migrate_registration_logs.py

```diff
@@ -0,0 +1,51 @@
+import logging
+
+from modularodm import Q
+
+from website.models import NodeLog
+from website.app import init_app
+
+from scripts import utils as script_utils
+from framework.transactions.context import TokuTransaction
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def get_targets():
+    logs = NodeLog.find(
+            Q('action', 'eq', 'embargo_approved') |
+            Q('action', 'eq', 'embargo_initiated') |
+            Q('action', 'eq', 'retraction_approved') |
+            Q('action', 'eq', 'retraction_initiated') |
+            Q('action', 'eq', 'registration_initiated')
+    )
+    return logs
+
+
+def migrate_log(logs):
+    for log in logs:
+        if log.node.registered_from:
```

Comment:
This PR's migration was done before https://github.com/CenterForOpenScience/osf.io/pull/4950, yes?  

Can you explain to me were the log.node field is coming from?  https://github.com/CenterForOpenScience/osf.io/pull/4950 added original_node and node fields.  Prior to that migration there were just nodes on the backrefs and a params['node'].  What is log.node then, seeing as it didn't exist yet?  Is it referencing a node on the backrefs?